core/tests/test_validations.py

```diff
@@ -11,9 +11,9 @@ def test_invalid_phone_number():
 
     # result = controller(
     #     Actions.ADD,
-    #     ContactPayload(name="Joe", phones={"12345678912345"}),
-    # )  # to long
-    # assert result.value == ValidationMessages.PHONE_NUMBER_LENGTH.value
+    #     ContactPayload(name="Joe", phones={"testString123"}),
```

Comment:
I guess we should have both of them (to long and unapropriate format)

Reply:
My mistake during merging, sorry. I'll fix it